src/Native/CMakeLists.txt

```diff
@@ -105,6 +105,10 @@ if (CLR_CMAKE_PLATFORM_UNIX)
     add_compile_options(-fvisibility=hidden)
 
     if(CLR_CMAKE_PLATFORM_DARWIN)
+        add_definitions(-DUNWIND_WORKAROUND)
```

Comment:
If the workaround is limited to a single place now, it is not very valueable to have it controlled via special define...